tools/rna_tools/rnacode/rnacode.xml

```diff
@@ -10,6 +10,15 @@
     <version_command>RNAcode --version</version_command>
     <command>
```

Comment:
Can we use `detect_errors` here and remove the stdio?

Reply:
should be fine